package.json

```diff
@@ -20,20 +20,29 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "react": "^0.14.8",
+    "react-dom": "^0.14.8",
+    "fxos-mvc": "fxos-eng/mvc#master",
     "babel-plugin-transform-es2015-modules-amd": "^6.6.5",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
-    "fxos-build": "^0.8.0",
-    "gulp": "^3.9.1",
-    "gulp-rename": "^1.2.2"
+    "alameda": "^1.0.0",
+    "gulp": "^3.9.1"
   },
```

Comment:
Did you install the dependencies with `npm install xxx --save`? If so, it should sort the dependencies in alphabetic order.